### PR TITLE
{bio} Bowtie-1.1.2-intel-2015a - Fix files to copy.

### PR DESCRIPTION
**Pull request by ehiggs**

### CODE REVIEW

File: easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-intel-2015a.eb [hunk 1]
```diff
@@ -21,7 +21,8 @@ patches = ['int64typedef.patch']
 
 buildopts = 'CC="$CC" CXX="$CXX" CPP="$CXX"'
 
-files_to_copy = [(['bowtie', 'bowtie-build', 'bowtie-inspect'], 'bin')]
+files_to_copy = [(['bowtie', 'bowtie-build', 'bowtie-inspect', 'bowtie-build-s', 'bowtie-build-l', 'bowtie-align-s', 
+    'bowtie-align-l', 'bowtie-inspect-s', 'bowtie-inspect-l', ], 'bin')]
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-intel-2015a.eb]: let's flesh this out in a list that we can reuse in the sanity check?

```python
suffixes = ['', '-build', '-inspect', '-build-s', '-build-l', '-align-s', '-align-l', '-inspect-s', '-inspect-l']
files_to_copy = [(['bowtie%s' % x for x in suffixes], 'bin')]

sanity_check_paths = {
    'files': ['bin/bowtie%s' % x for x in suffixes],
    'dirs': [],
}
```

**Comment by boegel** [comment]
@ehiggs: another option is to fix the existing `bowtie.py` easyblock, and use that instead of `MakeCp`...

I'm not sure why we're using `MakeCp` at all here, since we already have a custom easyblock for Bowtie...

We should probably use something like `glob.glob(os.path.join(srcdr, 'bowtie*'))` in the Bowtie easyblock.

**Comment by ehiggs** [comment]
I used MakeCp because v1.1.2 has been updated to use a more traditional approach.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/75de6c030c78df272c0b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b4449ab87215cecc789f for a full test report.

**Comment by boegel** [comment]
Going in, thanks for figuring this out @ehiggs!

**Pull request merged**